pandas/core/arrays/interval.py

```diff
@@ -212,7 +212,13 @@ def __new__(
 
     @classmethod
     def _simple_new(
-        cls, left, right, closed=None, copy=False, dtype=None, verify_integrity=True
+        cls,
+        left,
+        right,
+        closed=None,
+        copy=False,
+        dtype: Optional[Dtype] = None,
```

Comment:
by the time we get here we should have a DtypeObj shouldnt we?

Reply:
The `_simple_new` method is a.o. used in the `__new__` method on L169 of this file. The `__new__` method accepts the dtype values of type `Dtype` and passes  it without modification to the dtype arg of `_simple_new`. Therefore I also gave it type Dtype here.